packages/web/src/features/translation/TranslationView.tsx

```diff
@@ -461,6 +473,16 @@ export default function TranslationView() {
           }
         }}
       />
+      <TranslationProgressBar
```

Comment:
We should conditionally render this based on whether the user is a member of the language, so that normal readers don't see it

Reply:
Done.